pkg/gameserversets/controller.go

```diff
@@ -252,34 +256,46 @@ func (c *Controller) gameServerEventHandler(obj interface{}) {
 	gsSet, err := c.gameServerSetLister.GameServerSets(gs.ObjectMeta.Namespace).Get(ref.Name)
 	if err != nil {
 		if k8serrors.IsNotFound(err) {
-			c.logger.WithField("ref", ref).Info("Owner GameServerSet no longer available for syncing")
+			c.baseLogger.WithField("ref", ref).Info("Owner GameServerSet no longer available for syncing")
 		} else {
-			runtime.HandleError(c.logger.WithField("gs", gs.ObjectMeta.Name).WithField("ref", ref),
+			runtime.HandleError(c.baseLogger.WithField("gsKey", gs.ObjectMeta.Namespace+"/"+gs.ObjectMeta.Name).WithField("ref", ref),
 				errors.Wrap(err, "error retrieving GameServer owner"))
 		}
 		return
 	}
 	c.workerqueue.EnqueueImmediately(gsSet)
 }
 
+func loggerForGameServerSetKey(base *logrus.Entry, key string) *logrus.Entry {
+	return base.WithField(gameServerSetKeyFieldName, key)
```

Comment:
Should this not use `logfields.GameServerSetKey` ?

Reply:
fixed